### DISCUSSION TITLE
AIR for iOS & Android

### DISCUSSION THREAD

**Discussion by yozef** [airsdk/Adobe-Runtime-Support#1091] [Q&A]
I haven't used AIR for mobiles in over a decade, and focused solely for Desktop (Mac, Windows & Linux).

I jumped on other cross-platform SDKs for mobiles, however I still see developers using AIR for mobile Apps.

My question are:
- has the performance improved?
- As I recall, the AIR Apps didn't run natively, I suppose it is still the case?
- The UI components for iOS & Android we're not native UI components, is it still the case?
- Are AIR for mobile catered mostly for Games?

**Comment by GeneralVimes** [comment]
I can answer 4 and perhaps 1. Please, check my games, they are all made with AIR: https://play.google.com/store/apps/dev?id=6075165531768716801

For example, I've got this comment from a player:
https://twitter.com/GameGems/status/996853431708069888 (it's written "I wanted to make the game lag, but I failed, you devs are cool")

**Reply by yoshimitsuspeed** [reply]
@GeneralVimes 

I am trying to figure out how to get this game running on Android and hoping to find a way to port it or get it where people can easily install and run it. 
In trying to learn about Adobe Air for Android I came across your post. 
Would you have any interest in helping me try to find a way to get this running and easily installable on Android? 
https://www.myabandonware.com/game/people-power-the-game-of-civil-resistance-dwm#download
